server/api/product/product.model.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+var mongoose = require('mongoose'),
+    Schema = mongoose.Schema;
```

Comment:
We should follow our style guide by declaring each variable independently - no use of commas